client/src/resources/CurrentItemProvider.js

```diff
@@ -21,4 +22,9 @@ export default class CurrentItemProvider {
     }
     return item;
   }
+
+  getDataByPointer(pointer) {
+    const ptr = jsonPointer.compile(pointer);
```

Comment:
`jsonPointer.get(this.item.conf, pointer)` is currently more performant if yo don't cache the variable you get here.